packages/modeling/src/operations/booleans/measureEpsilon.js

```diff
@@ -0,0 +1,71 @@
+const flatten = require('../../utils/flatten')
+
+const { EPS } = require('../../math/constants')
+
+const { geom2, geom3, path2, poly3 } = require('../../geometry')
+
+const measureBounds = require('../measurements/measureBounds')
+
+/*
+ * Measure the epsilon of the given (path2) geometry.
+ * @return {Float} the epsilon (precision) of the geometry
+ */
+const measureEpsilonOfPath2 = (geometry) => {
+  let bounds = measureBounds(geometry)
+  let x = bounds[1][0] - bounds[0][0]
+  let y = bounds[1][1] - bounds[0][1]
+
+  let epsilon = (x + y) / 2 * EPS
+  return epsilon
+}
+
+/*
+ * Measure the epsilon of the given (geom2) geometry.
+ * @return {Float} the epsilon (precision) of the geometry
+ */
+const measureEpsilonOfGeom2 = (geometry) => {
+  let bounds = measureBounds(geometry)
+  let x = bounds[1][0] - bounds[0][0]
+  let y = bounds[1][1] - bounds[0][1]
+
+  let epsilon = (x + y) / 2 * EPS
+  return epsilon
+}
+
+/**
+ * Measure the epsilon of the given (geom3) geometry.
+ * @return {Float} the epsilon (precision) of the geometry
+ */
+const measureEpsilonOfGeom3 = (geometry) => {
+  let bounds = measureBounds(geometry)
+  let x = bounds[1][0] - bounds[0][0]
+  let y = bounds[1][1] - bounds[0][1]
+  let z = bounds[1][2] - bounds[0][2]
+
+  let epsilon = (x + y + z) / 3 * EPS
```

Comment:
Refering to Dick Gregorius Quick Hull (http://media.steampowered.com/apps/valve/2014/DirkGregorius_ImplementingQuickHull.pdf) page 30.

The epsilon implementation in the document is different form yours. Did you want to implement it the Dick Gregorius QuickHull way?
As reference implementation we may consider:
https://github.com/mrdoob/three.js/blob/219e3c1bc24163e2c31ab6f5cf4bb66f81b689bc/examples/jsm/math/ConvexHull.js#L495

Unfortunately, I could not find any background information why Dick Gregorius presents this formula or on what geometric/mathematical insight it is based :( 

I am totally fine with your implementation, just pointing out that it is different.

If you would like to change to the reference implementation I could adjust the code for you.

Reply:
The background is choosing edges (or faces) in the same plane. Very closely aligned edges should be considered within the same plane. Look at page 29.

Yeah. It’s a slightly different calculation but basically the same math. I used our constant EPS as the basis for 1 X 1 X 1. We know that this EPS works for designs up to 10 X 10 or more. Any larger design needs a larger EPS. And the same applies for really small designs.

The EPS being calculated in three.js hull is for checking equality of points. It’s slightly different then this use case. And way too small, really.

Reply:
Oh, I didn't see you were using a custom EPS value.

Unfortunately, I also have to confirm that calculations based on floating point epsilon yield way to small numbers. You are totally right.

Reply:
No problem. I like the feedback. It makes me THINK about what I’m trying to fix. :)